README.md

```diff
@@ -253,54 +253,48 @@ To add ZSS as a security provider, add the **sso-auth** plugin to the App Server
-You must also define or set **agent.https.port**. This is the TCP port which ZSS will listen on to be contacted by the App Server. Define this in the configuration file as a value between 1024-65535. See [zss configuration](https://github.com/zowe/zlux/wiki/Configuration-for-ZLUX-App-Server-&-ZSS#zss-configuration) for more information and an example.
-
-As a result of the above edits to server.json, an example of what it may now look like is:
-
-```json
-{
-  "node": {
-    "https": {
-      "ipAddresses": ["0.0.0.0"],
-      "port": 7556,
-      "keys": ["../defaults/serverConfig/zlux.keystore.key"],
-      "certificates": ["../defaults/serverConfig/zlux.keystore.cer"],
-      "certificateAuthorities": ["../defaults/serverConfig/apiml-localca.cer"]
-    },
-    "childProcesses": [
-      {
-        "path": "../bin/zssServer.sh",
-        "once": true
-      }
-    ]
-  },
-
-  "agent": {
-    "host": "localhost",
-    "https": {
-      "port": 7557
-    }
-  },
-
-  "productDir":"../defaults",
-  "siteDir":"~/.zowe/workspace/app-server/site",
-  "instanceDir":"~/.zowe/workspace/app-server",
-  "groupsDir":"~/.zowe/workspace/app-server/groups",
-  "usersDir":"~/.zowe/workspace/app-server/users",
-  "pluginsDir":"~/.zowe/workspace/app-server/plugins",
-
-  "dataserviceAuthentication": {
-    "defaultAuthentication": "saf",
-    "rbac": false
-  }
-}
+Within the zowe.yaml file, you need to define or set **components.zss.agent.agent.host** to a hostname or ip address where ZSS is located that is also visible to the App Server. This could be '0.0.0.0' or the hostname of a z/OS system.
+You must also define or set **components.zss.agent.agent.port**. This is the TCP port which ZSS will listen on to be contacted by the App Server. Define this in the configuration file as a value between 1024-65535. See [zss configuration](https://github.com/zowe/zlux/wiki/Configuration-for-ZLUX-App-Server-&-ZSS#zss-configuration) for more information and an example.
+
+As a result of the above edits to zowe.yaml, an example of what it may now look like is:
+
+```yaml
+components:
+  app-server:
+    node:
+      https:
+        ipAddresses:
+        - 0.0.0.0
+        port: 7556
+        keys:
+        - "../defaults/serverConfig/zlux.keystore.key"
+        certificates:
+        - "../defaults/serverConfig/zlux.keystore.cer"
+        certificateAuthorities:
+        - "../defaults/serverConfig/apiml-localca.cer"
+      childProcesses:
+      - path: "../bin/zssServer.sh"
+        once: true
+    productDir: "../defaults"
+    siteDir: "~/.zowe/workspace/app-server/site"
+    groupsDir: "~/.zowe/workspace/app-server/groups"
+    usersDir: "~/.zowe/workspace/app-server/users"
+    pluginsDir: "~/.zowe/workspace/app-server/plugins"
+    dataserviceAuthentication:
+      defaultAuthentication: saf
+      rbac: false
+  zss:
+    agent:
```

Comment:
Same here?

Reply:
This comment may be wrong because my components/app-server/share/zlux-app-server/defaults/serverConfig/zowe.yaml doesn't have enabled: true so as long as base zowe.yaml has it, it may be ok